src/baldr/graphtile.cc

```diff
@@ -47,6 +47,16 @@ std::string GenerateTmpSuffix() {
   return ss.str();
 }
 
+std::vector<std::string> split(const std::string& source, char delimiter) {
```

Comment:
this is no longer used. i think you can delete it

Reply:
@kevinkreiser ty. removed